docs/user-guide.md

```diff
@@ -137,7 +137,8 @@ strategies:
 Descheduler's `RemovePodsViolatingNodeTaints` strategy can be combined with
 [Node Problem Detector](https://github.com/kubernetes/node-problem-detector/) and
```

Comment:
It seems like NPD doesn't really do anything to help with autohealing, so we could probably just remove it entirely from this section. Or, add a note in here that it could eventually be combined here when https://github.com/kubernetes/node-problem-detector/pull/565 is available in NPD

Reply:
thanks, i add a note now :)